### ISSUE TITLE
Multiple Text Styling Issues

### ISSUE THREAD

**Issue by sxmawl** [fossasia/fossasia.org#827]
- ###  **Issue 1 - (Text Overflow on mobile screen)**

**Desktop**

This is the complete text that should be displayed.

<img width="1242" alt="Screenshot 2021-04-16 at 1 30 32 PM" src="https://user-images.githubusercontent.com/69129797/114992042-ed28a600-9eb7-11eb-898c-d9f3c1cb0155.png">

**Mobile**

On Mobile screens, the text in the section "**A little more about FOSSASIA**" overflows and gets hidden.

<img width="215" alt="Screenshot 2021-04-16 at 1 35 44 PM" src="https://user-images.githubusercontent.com/69129797/114992694-a9826c00-9eb8-11eb-94a8-72a2c8c2ddfe.png">

- ###  **Issue 2 - (Unreadability)**

The text color of names of apps such as **SUSI.AI, PSLab** and **Phimp.me** is merging with the background color and hence making it difficult to read

<img width="1126" alt="Screenshot 2021-04-16 at 1 19 40 PM" src="https://user-images.githubusercontent.com/69129797/114990697-68895800-9eb6-11eb-988b-bd4f50baae48.png">

They are positioning, overflow and text-color CSS changes. I'd love to work on this issue if someone can assign it to me.

**Comment by jasbirrajrana** [comment]
yeah, @SakshamDevelops seems to have issues with the responsiveness of the website!

**Comment by sxmawl** [comment]
Yea @jasbirrajrana, seems like couple of minor css changes. Would be happy to fix this, should I go ahead a create a PR?

**Comment by Shiva336** [comment]
I want to work on this issue. Could someone help me out?

**Comment by 2201monesh** [comment]
Hey I would like to work on it can you suggest me ways to get started.

**Comment by Gupta-Nikhil85** [comment]
Hey I would like to work on it.
Can anyone suggest me the way to get started.
